src/extension/iSamples_rangeFacet.js

```diff
@@ -2,7 +2,9 @@ import PropTypes from 'prop-types';
 import React from "react";
 import cx from "classnames";
 
-import RangeSlider from "./iSamples_rangeSlider";
+import {
+  defaultComponentPack
+} from "solr-faceted-search-react";
```

Comment:
@datadavev please have a look at this and see which you like better -- the corresponding change in `Solr-faceted-search-react` is https://github.com/isamplesorg/solr-faceted-search-react/pull/6/commits/5c4ebb1da6dda4eebc8cfcb7ee421d65c0c4b49c